.github/workflows/ci.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           RUSTFLAGS: -C debug-assertions
         with:
           command: test
-          args: ${{matrix.args}}
+          args: ${{matrix.args}} --features "internals"
```

Comment:
optional nit that can be come an issue for the future: we should probably find an efficient way to permutate all the possible combinations of features.